content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/count_products.rst

```diff
@@ -129,12 +129,16 @@ Count products
 ==============
 
 Counting products is a recurring activity in a warehouse. Once a count is complete, go to
-:menuselection:`Inventory app --> Operations --> Inventory Adjustments` to update the
+:menuselection:`Inventory app --> Operations --> Physical Inventory` to update the
 :guilabel:`Counted Quantity` column for each product line.
 
+To print a PDF of a count sheet, add products to the count sheet by ticking their checkboxes. Click
+:guilabel:`Count sheet` to download a PDF that contains the selected products.
```

Comment:
The count sheet button is not available by default; you have to click the Print button to get to it, first (:

![image](https://github.com/user-attachments/assets/f8392d5c-b687-40a1-a3e3-456ce1d8abdf)

Please tweak this line a little bit to have the documentation reflect *precisely* how things appear in the UI (:

Also! This a great opportunity to use the printer icon
```
:icon:`fa-print me-1`
```